featuretools/tests/entityset_tests/test_es.py

```diff
@@ -664,138 +663,6 @@ def test_checks_time_type_setting_secondary_time_index(es):
-        instance_ids=[1])
-    related = ["Haribo sugar-free gummy bears", "coke zero"]
-    assert set(related) == set(result['id'].values)
-
-
-def test_related_instances_all(es):
-    # test querying across the entityset
-    result = es.related_instances(
-        start_entity_id='customers', final_entity_id='products',
-        instance_ids=None)
-
-    for p in es['products'].df['id'].values:
-        assert p in result['id'].values
-
-
-def test_related_instances_all_cutoff_time_same_entity(es):
-    # test querying across the entityset
-    result = es.related_instances(
-        start_entity_id='log', final_entity_id='log',
-        instance_ids=None, time_last=pd.Timestamp('2011/04/09 10:30:31'))
-
-    assert result['id'].values.tolist() == list(range(5))
-
-
-def test_get_pandas_slice(es):
-    assert set(_slice_for(es, 'products').keys()), set(['products', 'log'])
-    assert set(_slice_for(es, 'customers').keys()) == set(
-        ['customers', 'sessions', 'log'])
-    assert set(_slice_for(es, u'régions').keys()) == set(
-        [u'régions', 'stores', 'customers', 'sessions', 'log'])
-
-    # make sure different subsets of the log are included in each filtering
-    assert set(_slice_for(es, 'customers')['log']['id'].values) == set(range(10))
-    assert set(_slice_for(es, 'products')['log']['id'].values) == set(
-        list(range(10)) + list(range(11, 15)))
-    assert set(_slice_for(es, u'régions')['log']['id'].values) == set(range(17))
-
-
-def test_get_pandas_slice_times(es):
-    # todo these test used to use time first time last. i remvoed and it
-    # still passes,but we should double check this okay
-    filter_eids = ['products', u'régions', 'customers']
-    start = np.datetime64(datetime(2011, 4, 1))
-    end = np.datetime64(datetime(2011, 4, 9, 10, 31, 10))
-
-    # make sure no times outside range are included in any frames
-    for eid in filter_eids:
-        result = _slice_for(es, eid, time_last=end)
-        for t in result['log']['datetime'].values:
-            assert t >= start and t < end
-
-        # the instance ids should be the same for all filters
-        for i in range(7):
-            assert i in result['log']['id'].values
-
-
-def test_get_pandas_slice_times_include(es):
-    # todo these test used to use time first time last. i remvoed and it
-    # still passes,but we should double check this okay
-    filter_eids = ['products', u'régions', 'customers']
-    start = np.datetime64(datetime(2011, 4, 1))
-    end = np.datetime64(datetime(2011, 4, 9, 10, 31, 10))
-
-    # make sure no times outside range are included in any frames
-    for eid in filter_eids:
-        result = _slice_for(es, eid, time_last=end)
-        for t in result['log']['datetime'].values:
-            assert t >= start and t <= end
-
-        # the instance ids should be the same for all filters
-        for i in range(7):
-            assert i in result['log']['id'].values
-
-
-def test_get_pandas_slice_secondary_index(es):
```

Comment:
i think this might be the only place in the tests we explicitly tested that secondary time index correctly filtered data.

a quick fix would be to add a test call to `Entity.query_by_values` that replicates this test.